### PR TITLE
doc: add troubleshooting steps for ceph-fuse

### PR DESCRIPTION
**Pull request by ktdreyer**
Thanks to @jcsp on [ceph-devel](http://thread.gmane.org/gmane.comp.file-systems.ceph.devel/32392/focus=32393) for this information.

### CODE REVIEW

File: doc/rados/troubleshooting/log-and-debug.rst [hunk 1]
```diff
@@ -138,6 +138,15 @@ Valgrind is computationally expensive, and will slow down your system otherwise.
 Valgrind messages are logged to ``stderr``. 
 
 
+ceph-fuse
+=========
+
+For ceph-fuse, try running in the foreground with logging to the
+console (``-d``) and enabling client debug (``--debug-client=20``), enabling
+prints for each message sent (``--debug-ms=1``) and debugging for the
+monitor client (``--debug-monc=20``).
```
Comment (jcsp) [inline doc/rados/troubleshooting/log-and-debug.rst]: I wouldn't generally advise monc=20 for client debugging, it was just in that particular case that it seemed like a potential mon issue.

File: doc/rados/troubleshooting/log-and-debug.rst [hunk 2]
```diff
@@ -138,6 +138,15 @@ Valgrind is computationally expensive, and will slow down your system otherwise.
 Valgrind messages are logged to ``stderr``. 
 
 
+ceph-fuse
```
Comment (jcsp) [inline doc/rados/troubleshooting/log-and-debug.rst]: Let's move this to cephfs/troubleshooting.rst

**Comment by jcsp** [comment]
ping @ktdreyer

**Comment by ktdreyer** [comment]
Thanks @jcsp for the review. I've updated my branch with your suggestions.

**Pull request merged**